discovery/wh_discovery.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 type webhookResponse struct {
-	Address string  `json:"address,omitempty"`
+	Address string  `json:"address"`
 	Score   float32 `json:"score,omitempty"`
```

Comment:
Should we also make `score` required?

Reply:
I'd leave it as `omitempty`. Would need to analyze more how it's used and if we always need it.